File: sample_recipe/bolezaixian.py
# !/usr/bin/env python
# -*- coding:utf-8 -*-

from feed_base import Feed

class BoLeZaiXian(Feed):
    name = 'bolezaixian'
    url = r'http://web.jobbole.com/feed/'
    capture = {}
    capture["catch"] = ['.entry']
    capture["remove"] = ['.copyright-area','#single-page-inner-widget','.post-adds','#author-bio','.crayon-main']

recipe = BoLeZaiXian

File: sample/bolezaixian.js

const Base = require('../common/recipe/feed2.js')

module.exports = class Feed extends Base{
  init(){
    super.init()
    this.name = '伯乐在线Web'
    this.url = 'http://web.jobbole.com/feed/'
    this.capture.catch = ['.entry']
    this.capture.remove = ['.copyright-area','#single-page-inner-widget','.post-adds','#author-bio','.crayon-main']
  }
}
